templates/managed-apps/index.html

```diff
@@ -82,7 +82,7 @@ <h2 class="p-muted-heading u-align--center">Our app and cloud experience lets yo
             loading="lazy",
             attrs={"class": "p-inline-images__logos"},
             ) | safe
-          }}
+          }}Find out more
```

Comment:
I don't think this should be here

```suggestion
          }}
```